java/lang/String/String.cpp

```diff
@@ -845,4 +850,170 @@ boolean String::operator>=(const String &target) const {
 		return false;
 	}
 	return true;
+}
+
+String String::print(const String &format, short value)

```

Comment:
Please add test case for String String::print(const String &format, short value) in TEST(JavaLang, StringFormat)